addons/barcodes/static/tests/barcode_scan.test.js

```diff
@@ -0,0 +1,32 @@
+/** @odoo-module **/
+
+import { describe, expect, test } from "@odoo/hoot";
+import { waitFor } from "@odoo/hoot-dom";
+import { Component, xml } from "@odoo/owl";
+import { BarcodeScanner } from "@barcodes/components/barcode_scanner";
+import { contains, mountWithCleanup } from "@web/../tests/web_test_helpers";
+
+describe.current.tags("desktop");
```

Comment:
Mmh :thinking: 
Barcode is more a use for mobile device.
There is also a `a.o_mobile_barcode` query in the test :smile: